src/hdrgen.d

```diff
@@ -2459,7 +2454,15 @@ public:
 
     override void visit(DeclarationExp e)
     {
-        e.declaration.accept(this);
+        if (auto v = e.declaration.isVarDeclaration())
+        {
+            buf.writeByte('(');
+            visitVarDecl(v, false);
+            buf.writeByte(';');
+            buf.writeByte(')');
```

Comment:
For header generation, a `DeclarationExp` is always printed via `ExpStatement`. If `DeclarationExp.toChars()` is directly called, it occurs by our debugging (e.g.: `printf("exp = %s\n", e.toChars());`).

Yes' the print format is a little strange, but during debugging, the unfitness to regular D code is rather good to recognize `DeclarationExp`.